monitoring-samples/README.md

```diff
@@ -0,0 +1,28 @@
+# Spring Cloud Data Flow Monitoring - Prometheus RSocket Proxy Architecture
+
+The Data Flow metrics architecture is designed around the [Micrometer](https://micrometer.io/) library - vendor-neutral application metrics facade.
+The Spring Cloud Data Flow server leverages the [Prometheus RSocket Proxy](https://github.com/micrometer-metrics/prometheus-rsocket-proxy), 
+for monitoring of `tasks`, which are short lived, as well as long lived `stream` applications. 
+The RSocket based monitoring architecture is reusable across the Local, Kubernetes and Cloud Foundry platforms. 
+
+The prebuilt `stream` and `task` applications support the Prometheus Proxy RSocket monitoring architecture. 
+
+The [stream-apps](./stream-apps/) samples shows how to enable monitoring for custom built `source`, `processor` and `sink` apps.  
+
+The [task-apps](./task-apps/) sample shows how to enable monitoring for custom built `task` apps.
+
+The following image shows the general architecture of how the Stream applications are monitored:
+
+![Stream App Monitoring](./images/SCDF-monitoring-prometheus-streams.png)
+
+and the same architecture is used for monitoring tasks as wel. 
+ 
+![Stream App Monitoring](./images/SCDF-monitoring-prometheus-tasks.png)
```

Comment:
"Grafana links shortcuts" <- not sure I understand that .. just describe it as a clickable button instead

Reply:
"clickable button" is bad as well.   What about "UI interface to access the SCDF dashboards" or just to leave it empty?

Reply:
Yeah, empty would be better.